stone/api.py

```diff
@@ -96,7 +96,7 @@ class ApiNamespace(object):
     """
 
     def __init__(self, name):
-        # type: (str) -> None
+        # type: (typing.Text) -> None
```

Comment:
FYI, `# type: (str)` may not play well in Python 2 with `from __future__ import unicode_literals`. Jus' sayin'.